src/org/labkey/targetedms/TargetedMSModule.java

```diff
@@ -202,7 +202,7 @@ public String getName()
     @Override
     public double getVersion()
     {
-        return 19.30;
+        return 19.33;
```

Comment:
Will need to adjust these numbers since I just merged the sample-scoped chromatograms branch